.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,88 @@
+name: Bug or Issue Report
+description: Report a bug or issue with GEOS-Chem
+title: "Put a one-line description here"
+labels: ["category: Bug"]
+body:
+  - type: input
+    id: name
+    attributes:
+      label: Your name
+    validations:
+      required: true
+  - type: input
+    id: affiliation
+    attributes:
+      label: Your affiliation
+    validations:
+      required: true
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened? What did you expect to happen?
+      placeholder: Describe your issue
+    validations:
+      required: true
+  - type: textarea
+    id: steps-to-reproduce
+    attributes:
+      label: What are the steps to reproduce the bug?
+      placeholder: Explain what you did
+    validations:
+      required: true
+  - type: textarea
+    id: log-files
+    attributes:
+      label: Please attach any relevant configuration and log files.
+      description: Drag and drop files like the GEOS-Chem log file, geoschem_config.rc, and HEMCO_Config.rc here. You may need to rename files to end in .txt first.
+    validations:
+      required: false
+  - type: input
+    id: version
+    attributes:
+      label: What GEOS-Chem version were you using?
+      placeholder: ex. 14.4.0
+    validations:
+      required: true
+  - type: dropdown
+    id: machine
+    attributes:
+      label: What environment were you running GEOS-Chem on?
+      multiple: true
+      options:
+        - Local cluster
+        - AWS
+        - Personal computer
+        - Other (please explain below)
+    validations:
+      required: true
+  - type: dropdown
+    id: compiler
+    attributes:
+      label: What compiler were you using?
+      multiple: true
+      options:
+        - Intel
+        - GNU
+        - PGI
```

Comment:
You may want to say `PGI (GEOS-Chem in CESM only)`

@lizziel would know the other compilers used by CESM

Reply:
Also would be nice to have a text box where they can type in the compiler version.

Reply:
I would also list GNU first, so that they are in alphabetical order

Reply:
In commit https://github.com/geoschem/geos-chem/pull/2288/commits/9a12d4819a4235dce35a0b7267eaac0e33f2727a, I removed the compiler dropdown and instead provide a text box for users to specify the compiler and version used

<img width="487" alt="Screenshot 2024-05-21 at 2 18 07 PM" src="https://github.com/geoschem/geos-chem/assets/4805437/b0adae41-d6e0-48e2-ac8d-43ca1d119226">

Reply:
We are referring CESM users to report issues at the CESM forum. I think we should limit the compilers listed to be for the offline model.

Reply:
The text box is fine.  Thanks @msulprizio